### PR TITLE
[AFD] Adds Mghost

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Adds mghost

## Why It's Good For The Game
I think we can trust mentors at this point, this helps them complete tickets properly

https://github.com/ParadiseSS13/Paradise/assets/108773801/a3ac0c1a-4978-4ea5-bd37-209b9f5659ad

## Testing
See above

## Changelog
:cl:
add: [MENTOR] Adds Mghost
/:cl:

### CODE REVIEW

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -387,6 +388,22 @@ GLOBAL_LIST_INIT(view_runtimes_verbs, list(
...
+		return
+
+	if(isobserver(mob))
+		to_chat(src, "You're already a ghost!")
+		return
+	else if(isnewplayer(mob))
+		to_chat(src, "You can't ghost while you're still in the lobby! Join or observe first.")
+	else
+		to_chat(src, "You are now a ghost!")
+		mob.gib()
```
Comment (FunnyMan3595) [inline code/modules/admin/admin_verbs.dm:405]:
```suggestion
		mob.gib()
		to_chat(src, "You are now a ghost!")
```

SMH, can't believe you would make such a mistake.

**Comment by Protointerwebs** [comment]
As a mentor I must say this is a mistake and I will abuse it in order to get the number 1 victory royal.

File: code/modules/admin/admin_verbs.dm (hunk 1 above)
Comment (Contrabang) [inline code/modules/admin/admin_verbs.dm:405]:
```suggestion
	if(isnewplayer(mob))
		to_chat(src, "You can't ghost while you're still in the lobby! Join or observe first.")
		return
	to_chat(src, "You are now a ghost!")
	mob.gib()
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by Burzah — APPROVED** [review]

**Comment by Burzah** [comment]
Good to go after Contra's review.

File: code/modules/admin/admin_verbs.dm [hunk 2]
```diff
@@ -387,6 +388,22 @@ GLOBAL_LIST_INIT(view_runtimes_verbs, list(
...
 		SSblackbox.record_feedback("tally", "admin_verb", 1, "Aghost") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/mentor_ghost()
+	set category = "Admin"
+	set name = "Mghost"
+
+	if(!check_rights(R_MENTOR))
+		return
+
+	if(isobserver(mob))
```
Comment (Contrabang) [inline code/modules/admin/admin_verbs.dm:398]:
```suggestion
	if(!isliving(mob))
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Comment by Contrabang** [comment]
image

File: code/modules/admin/admin_verbs.dm [hunk 3]
```diff
@@ -387,6 +388,22 @@ GLOBAL_LIST_INIT(view_runtimes_verbs, list(
...
+
+	if(!check_rights(R_MENTOR))
+		return
+
+	if(!isliving(mob))
+		to_chat(src, "You're already a ghost!")
+		return
+	if(isnewplayer(mob))
+		to_chat(src, "You can't ghost while you're still in the lobby! Join or observe first.")
+		return
```
Comment (Contrabang) [inline code/modules/admin/admin_verbs.dm:403]:
```suggestion
	if(isnewplayer(mob))
		to_chat(src, "You can't ghost while you're still in the lobby! Join or observe first.")
		return
	if(!isliving(mob))
		to_chat(src, "You're already a ghost!")
		return
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by Contrabang — APPROVED** [review]

**Comment by Coolrune206** [comment]
I'm disappointed this doesn't immediately kill you upon use.

**Pull request merged**